core/base/README.md

```diff
@@ -0,0 +1,7 @@
+# core-base
+
+## Configuration settings
+
+| Parameter name | Description        | Mandatory | Default value |
+|----------------|--------------------|-----------|---------------|
+| `edc.hostname` | Connector hostname | false     | localhost     |
```

Comment:
Could we put a little more information here so that users who are not looking at the code understand what this configuration is used for?

Maybe something like this:

```suggestion
| `edc.hostname` | Connector hostname, which e.g. is used in referer urls | false     | localhost     |
```

Reply:
seems fair, done